cmd/internal/apidump/apidump.go

```diff
@@ -201,6 +219,28 @@ func init() {
 		"",
 		"Your Akita project. Exactly one of --out or --project must be specified.")
 
+	Cmd.Flags().StringVar(
+		&postmanCollectionID,
+		"collection",
+		"",
+		"Your Postman collectionID. Both --collection and --key must be specified.")
+
+	Cmd.MarkFlagsMutuallyExclusive("out", "project", "collection")
+
+	Cmd.Flags().StringVar(
+		&postmanAPIKey,
+		"key",
+		"",
+		"Your Postman API Key. Both --collection and --key must be specified.")
+
+	Cmd.MarkFlagsRequiredTogether("collection", "key")
+
+	Cmd.Flags().StringVar(
+		&postmanEnvironment,
+		"environment",
+		"",
+		"Your Postman Environment. Default value is PREVIEW")
+
```

Comment:
I think we should probably make this hidden, as it will be irrelevant to all but Postman developers.

We should figure out when the cutover is from PREVIEW being the default, to production.  We can do that in a CLI release after 8/14, PREVIEW is probably best now.

Reply:
Another option is to make PREVIEW the default is domain is staging.akita.software and use PRODUCTION otherwise.

Reply:
Since this environment flag is only for running in lower envs not in production, I have decided to pass the environment header only if this argument is provided, otherwise, it will default to PREVIEW in backend ([ref](https://github.com/akitasoftware/superstar/blob/master/services/front/app/auth.py#L164)).

We can make the default postman env value config driven in BE and set the value of the config accordingly.

Reply:
We can remove `Default value is PREVIEW` message to remove confusion.

Reply:
`--domain` flag is still something we need to resolve